github/update-doxygen-doc.sh

```diff
@@ -299,28 +310,20 @@ mkdir -p doxygen/${doc_version}
 cp -rf ${build_dir}/doc/doxygen-html/* doxygen/${doc_version}/ \
  || abort "failed to copy the documentation"
 
-if test -x "$tmp/project/doxygen/$doc_version/installdox"; then
-    echo "* Run installdox to fix documentation cross-links..."
-
-    cd $tmp/project/doxygen/$doc_version/ || abort "Failed to change directory."
-
-    # Patch installdox to never abort.
-    # installdox normally checks that _all_ links are substituted.
-    # This is not what we want, we call installdox one dependency at a time
-    # to avoid resolving documentation dependencies manually.
-    sed 's|&usage();||g' installdox > installdox.fixed
-    chmod 755 installdox.fixed
-
-    i=1
-    for doxytag in $installdox_args; do
-	url=`echo "$installdox_urls" | cut -d' ' -f$i`
-	./installdox.fixed -q -l ${doxytag}.doxytag@$url \
-	    2> /dev/null > /dev/null \
-	    || true
-	i=$((i+1))
-    done
-    cd $tmp/project || abort "Failed to change directory."
-fi
+echo "* Fixing cross links..."
+cd "$tmp/project/doxygen/$doc_version"
+for package in ${GIT_DEPENDENCIES}
+do
+  git_dependency_parsing $package
+  if [ $git_dep_github = true ]; then
+    doxygen_url="https://${git_dep_organization}.github.io/${git_dep_project}/doxygen/HEAD/"
```

Comment:
Do you think it would be possible to support other domains than `organization.github.io` ? I understand this will be the most common use-case, but if someone wishes to either host the documentation elsewhere (readthedocs) or simply to use the top-level URL (For example `robotpim.net` points to `roboptim.github.io`)